docusaurus/docs/create-a-plugin/develop-a-plugin/set-up-github-workflows.md

```diff
@@ -22,11 +22,15 @@ The CI (`ci.yml`) workflow is designed to lint, type check, and build the fronte
 
 ## The release workflow
 
+The release (`release.yml`) workflow is designed to create a new release of your plugin whenever you're ready to publish a new version to Grafana Cloud. This automates the process of creating releases in GitHub and provides instructions for submitting the plugin to the Grafana plugin catalog.
```

Comment:
**Nit:** I don't think this is 100% correct, as what the workflow is doing is that it updates a signed and packed version of the plugin as an artifact - this can also be useful, if someone doesn't want to ship it for Grafana Cloud:
```
(...) whenever you're ready to publish a new version to Grafana Cloud.
```

(I know this was not introduced in this PR, just saw it.)

Reply:
good catch, will fix

Reply:
done, PTAL 🙏